acceptance/Rakefile

```diff
@@ -4,29 +4,31 @@ require 'rototiller'
 
 desc "Generate Beaker Host config"
 rototiller_task :host_config do |task|
-  task.add_env(name: 'BOLT_CONTROLLER', default: 'centos7-64')
-  task.add_env(name: 'BOLT_NODES',
-               default: 'ubuntu1604-64,osx1012-64,windows10ent-64')
-  ns = [ENV['BOLT_CONTROLLER'], ENV['BOLT_NODES']].join(',')
-  n  = ns.split(',')
-  n_new = []
-  n.each_with_index do |node, i|
-    roles = []
-    roles << 'bolt' if i.zero?
-    roles << if /win/ =~ node
-               'winrm'
-             else
-               'ssh'
-             end
-    node = node + roles.join(',') + '.{type=aio}'
-    n_new << node
+  unless ENV['BEAKER_HOSTS']
```

Comment:
Is it `BEAKER_HOSTS` or `BEAKER_HOSTS_FILE`? It seems to be `BEAKER_HOSTS_FILE` below.